test/fixtures/api_actions.yml

```diff
@@ -93,10 +93,9 @@ create_api_action_plugin_subdomain_events:
   type: CreateApiAction
   action_type: send_web_request
   include_api_resource_data: true
-  payload_mapping: >
-    {
-      "content": "#{self.representation.body}"
-    }
+  payload_mapping: {
```

Comment:
@Pralish what's the change here? We are turning JSON strings into ruby hashes? or is it something else

Reply:
@donrestarone , Yes, we are turning JSON strings into ruby hashes. The '>' was creating json string.